core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/OpenStackCredentialParam.java

```diff
@@ -6,7 +6,13 @@
 
     USER("user", true, String.class, Optional.<String>absent()),
     PASSWORD("password", true, String.class, Optional.<String>absent()),
-    TENANT_NAME("tenantName", true, String.class, Optional.<String>absent()),
+    TENANT_NAME("tenantName", false, String.class, Optional.<String>absent()),
+    USER_DOMAIN("userDomain", false, String.class, Optional.<String>absent()),
+    KEYSTONE_VERSION("keystoneVersion", true, String.class, Optional.<String>absent()),
+    KEYSTONE_AUTH_SCOPE("keystoneAuthScope", false, String.class, Optional.<String>absent()),
```

Comment:
KEYSTONE_AUTH_SCOPE can be only `cb-keystone-v3-default-scope` `cb-keystone-v3-project-scope` `cb-keystone-v3-domain-scope`